python/pyspark/util.py

```diff
@@ -107,6 +107,22 @@ def majorMinorVersion(sparkVersion: str) -> Tuple[int, int]:
             )
 
 
+class LogUtils:
```

Comment:
@HyukjinKwon Is there a better file location to put this constant string? There won't be a lot of new code in LogUtils.

Reply:
Yeah, I think so

Reply:
@HyukjinKwon where should it be?